src/ngx_http_lua_socket_tcp.c

```diff
@@ -6024,8 +6026,13 @@ ngx_http_lua_tcp_queue_conn_op_cleanup(void *data)
                    "lua tcp socket abort queueing, conn_op_ctx: %p, u: %p",
                    conn_op_ctx, u);
 
+#if (nginx_version >= 1007005)
     if (conn_op_ctx->event.posted) {
-        ngx_delete_posted_event(&conn_op_ctx->event);
+#else
+    if (conn_op_ctx->event.prev) {
+#endif
+        ev = &conn_op_ctx->event;
```

Comment:
Why introduce the extra `ev` variable? It's only used once.

Reply:
@agentzh Thanks for the feedback, it has been fixed.